.github/workflows/build.yml

```diff
@@ -13,10 +13,23 @@
 # limitations under the License.
 name: build
 
-on: ['push']
+on:
+  create:
+  push:
+    branches:
+      - 'master'
+      - 'v*'
+    paths-ignore:
+      - '*.md'
+      - 'changelogs/**'
+      - 'example/**'
```

Comment:
Is the example directory used for any testing purpose or is it purely for examples?

Reply:
The files for testing are in script/ directory. example/ hasn't been used in any of the GitHub Action workflows.